src/app/core/core.module.ts

```diff
@@ -52,6 +53,7 @@ import { DimensionDirective } from './directives/dimension.directive';
     NzIconModule,
     NzSelectModule,
     NzRadioModule,
+    RouterModule,
   ],
   entryComponents: [],
   // providers: [{ provide: ApiService, useClass: MockApiService }],
```

Comment:
delete this

Reply:
Navbar need to be have 'navigate to home' function